force_wfmanager/wfmanager_task.py

```diff
@@ -68,16 +74,19 @@ class WfManagerTask(Task):
             TaskAction(
                 name='Open Workflow...',
                 method='open_workflow',
+                enabled_name='_ui_enabled',
                 accelerator='Ctrl+O',
             ),
             TaskAction(
                 name='Save Workflow',
                 method='save_workflow',
+                enabled_name='_ui_enabled',
                 accelerator='Ctrl+S',
             ),
             TaskAction(
                 name='Save Workflow as...',
                 method='save_workflow_as',
+                enabled_name='_ui_enabled',
```

Comment:
Not for an action apparently, I have to define a name which refers to a Boolean trait